kotlin-asyncapi-core/src/test/kotlin/org/openfolder/kotlinasyncapi/model/AsyncApiIntegrationTest.kt

```diff
@@ -79,10 +79,8 @@ internal class AsyncApiIntegrationTest {
                                 }
                             }
                         }
-                        bindings {
-                            reference("http") {
-                                ref("#/components/messageBindings/streamingHeaders")
-                            }
+                        bindingsRef {
+                            ref("#/components/messageBindings/streamingHeaders")
```

Comment:
Just for my understanding, we will not depend on OpenApi specs anymore for the components part right?

Reply:
where did we depend on openapi before?